src/ResolvedPeers.cc

```diff
@@ -227,3 +225,19 @@ operator <<(std::ostream &os, const ResolvedPeers &peers)
     return os << peers.size() << (peers.destinationsFinalized ? "" : "+") << " paths";
 }
 
+/* PeerConnectionPointer */
+
```

Comment:
```suggestion
```

Reply:
This comment has more reason to exist, since IMO it does declare the code following to be misplaced. Serving the purpose Alex cited for using these still. So I do not insist on this one being removed.

@eduard-bagdasaryan, up to you on this one.

Reply:
Left as is.